src/model/wallet.ts

```diff
@@ -191,10 +191,11 @@ export const walletInit = async (
   name = null,
   overwrite = false,
   checkedWallet = null,
-  network: string
+  network: string,
+  defaultIsNew: boolean = false
 ): Promise<WalletInitialized> => {
   let walletAddress = null;
-  let isNew = false;
+  let isNew = defaultIsNew;
```

Comment:
instead of `defaultIsNew`, we could set `isNew`'s initial value to the seedphrase check.